.devcontainer/Dockerfile

```diff
@@ -148,10 +148,9 @@ RUN --mount=type=cache,target=/root/.cache/pip/,id=pip-cache \
   set -eux \
   && echo "Installing build system packages" \
     && apt-get update \
-    && apt-get install --yes --quiet ${BUILD_PACKAGES}
-
-RUN set -eux \
-  && npm update npm -g
+    && apt-get install --yes --quiet ${BUILD_PACKAGES} \
+    && npm update npm -g \
+    && npm install -g @angular/cli@~18.2.11
```

Comment:
This doesn't look correct, npm install is not "installing build system packages". Also pinning this to a specific version of the angular cli seems very annoying

Reply:
removed with this commit "remove angular cli installation from Dockerfile"

Reply:
Thanks. why the changes here at all?

Reply:
For new contributors to this project I think it's a kind of easyier to start with devcontainers. All you have to do it's simply open the cloned project in vscode and wait for the dev environement to be ready for debugging purposes. This avoids potential missing packages to be installed on you host system. 
Running `pipenv install --dev` i get the errors here below where as with devcontainer i do not have worry about host issues :-)

<img width="1019" alt="image" src="https://github.com/user-attachments/assets/9097a9e0-4881-4478-b78d-dc8fe5e7cc9e" />

Reply:
No sorry I just meant why the changes to these lines in the dockerfile

Reply:
initally i thaught i should install the angular cli within the Dockerfile and finaly your comment made me realise that indeed the angluar cli has not to be installed by the Dockerfile